netutils/nng/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config NETUTILS_NNG
+	tristate "Enable Nanomsg NG"
```

Comment:
tristate to bool, tristate is only meaningful for program(builtin v.s. elf file).